content/06.quality_criteria.md

```diff
@@ -382,10 +396,12 @@ public repository of such artifacts and notification of the success of the proce
 * **[QC.Del03]** Upon success of the previous (**QC.Del02**) process, a notification **MUST** be
   sent to pre-defined parties such as the main developer or team.
 
-## 4.15. Automated Deployment [QC.Dep]
+## 4.6. Deployment
```

Comment:
Same comment here, if we think that deploy is part of a change management, then this could be merged with "Code management"

Reply:
I would keep this in the last section, as a logical thing, eventually you may not always deploy after every change